units/heroes/1.12.cfg

```diff
@@ -0,0 +1,396 @@
+    [death]
+        start_time=0
+        [frame]
+            image="units/undead/{BASE_NAME}-die-[1~4].png:150,units/undead/soulless-die-[5~10].png:150"
+        [/frame]
+    [/death]
+    [attack_anim]
+        [filter_attack]
+            name=touch
+        [/filter_attack]
+        direction=s
+        start_time=-200
+        [frame]
+            image="units/undead/{BASE_NAME}-attack-s.png:400"
+            sound=zombie-attack.wav
+        [/frame]
+    [/attack_anim]
+    [attack_anim]
+        [filter_attack]
+            name=touch
+        [/filter_attack]
+        direction=n
+        start_time=-200
+        [frame]
+            image="units/undead/{BASE_NAME}-attack-n.png:400"
+            sound=zombie-attack.wav
+        [/frame]
+    [/attack_anim]
+    [attack_anim]
+        [filter_attack]
+            name=touch
+        [/filter_attack]
+        direction=se,sw,ne,nw
+        start_time=-200
+        [frame]
+            image="units/undead/{BASE_NAME}-attack.png:400"
+            sound=zombie-attack.wav
+        [/frame]
+    [/attack_anim]
+#enddef
+
+
+
+
+[unit_type]
+    id=Soulless Hero
+    #textdomain wesnoth-units
+    name= _ "Soulless"
+    do_not_list=yes
+    profile="portraits/zpeasant.png"
+    small_profile="portraits/zpeasant.png~SCALE(205,205)"
+    race=undead
+    {TRAIT_FEARLESS_MUSTHAVE}
+    {MAGENTA_IS_THE_TEAM_COLOR}
+    experience=35
+    level=1
+    alignment=chaotic
+    advances_to=Running Hero
+    {AMLA_DEFAULT}
+    cost=13
+    usage=fighter
+    description= _ "The technique of animating a dead body is unfortunately well-known in the dark arts; practitioners often use it to raise servants and soldiers from unwilling corpses. These shamblers are often numerous, but fragile; a touch of combat can waken them, though, making them far more formidable."+{SPECIAL_NOTES}+{SPECIAL_NOTES_PLAGUE}
+    #textdomain wesnoth-Danse_Macabre
+    die_sound={SOUND_LIST:ZOMBIE_HIT}
+
+    [attack]
+        name=touch
+        description={STR_TOUCH}
+        icon=attacks/touch-zombie.png
+        type=impact
+        range=melee
+        damage=7
+        number=3
+        [specials]
+            {WEAPON_SPECIAL_PLAGUE}
+        [/specials]
+    [/attack]
+
+    [resistance]
+        arcane=140
+    [/resistance]
+    [movement_costs]
+        deep_water=4
+    [/movement_costs]
+    [defense]
+        deep_water=90
+    [/defense]
+
+    {UNIT_BODY_SOULLESS_STATS smallfoot 4 28}
+    {UNIT_BODY_SOULLESS_GRAPHICS soulless}
+
+    [variation]
+        variation_id=gryphon
+        variation_name= _ "Pech"
+        inherit=yes
+        profile="portraits/zPech.png"
+        small_profile="portraits/zPech.png~SCALE(205,205)"
+        {UNIT_BODY_SOULLESS_STATS    fly 8 33} # +3 MP
+        {UNIT_BODY_SOULLESS_GRAPHICS soulless-drake}
+        gender=female
+    [/variation]
+
+    [variation]
+        variation=id=saurian
```

Comment:
Should be variation_id. As is now, we cannot create the variation, though it is present in help.